### DISCUSSION TITLE
the Value type of rendered template?

### DISCUSSION THREAD

**Discussion by cataggar** [mitsuhiko/minijinja#342] [Q&A]
Is it possible to get the Value type of the rendered template? For example, a template of `{{ 1 + 2 }}` renders `"3"`. I would like to know it if it evaluates to a number.

**Comment by cataggar** [comment]
I posted code to https://github.com/mitsuhiko/minijinja/pull/343 that should explain better.

**Comment by mitsuhiko** [comment]
To copy the answer to the closed issue here:

A template always returns a string. If you want to know that `1 + 3` was an integer, you need to use expression evaluation instead and avoid the use of `{{` / `}}`. There is no guarantee in the engine that the information of the last operation is retained.
